packages/angular-test/src/text.ts

```diff
@@ -10,11 +10,13 @@ export const textTest = <C extends JsonFormsControl>(
     imports: any[],
     providers: any[],
     componentUT: Type<C>,
-    errorTestInformation: ErrorTestInformation) => () => {
+    instance: string,
+    errorTestInformation: ErrorTestInformation,
+    elementToUse: (element: DebugElement) => any) => () => {
         let fixture: ComponentFixture<any>;
-        let textNativeElement: HTMLInputElement;
-        let component: C;
         let textElement: DebugElement;
+        let textNativeElement;
+        let component: C;
 
         const data = { foo: 'foo' };
         const schema: JsonSchema = {
```

Comment:
I missed this before, but on line 26 foo has a type of 'text', which should be 'string', I think. Which brings me to the question whether this `schema` has any actual effect on the tests. Can we clarify that?

Reply:
should be fixed with 2 commit